Example/Info.plist

```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.5.2</string>
+	<string>2.5.3</string>
 	<key>CFBundleVersion</key>
-	<string>110</string>
+	<string>111</string>
```

Comment:
`111` was used for `2.6.0-beta.2`, so consider changing it to the latest (`119` as far as `2.8.0-alpha.1`).

Reply:
Actually I do not think that we take into account bundle versions of another releases when releasing minor ones (e.g. `v2.5.0-beta.2` and `v2.4.1` have similar bundle version).

Reply:
Looking at where `CFBundleVersion` could be useful, I can think of determining different flavors of one release (alpha, beta, rc, final). So in the context of different releases, we may not worry indeed.